addons/account/models/chart_template.py

```diff
@@ -269,7 +269,7 @@ def tax_template_changed(tax, template):
                         if (
                             command not in (Command.UPDATE, Command.CREATE)
                             or not self.ref(vals['tax_src_id'], raise_if_not_found=False)
-                            or not self.ref(vals['tax_dest_id'], raise_if_not_found=False)
+                            or not self.ref(vals.get('tax_dest_id', ""), raise_if_not_found=False)
```

Comment:
```suggestion
                            or not self.ref(vals.get('tax_dest_id'), raise_if_not_found=False)
```

Reply:
I think we should pass at least an empty string as the `self.ref` expects a string. What do you say?

Reply:
But `self.ref` doesn't
https://github.com/odoo/odoo/blob/ca62e32aa4bca26c4604eeb2d004a230eeff0c0d/addons/account/models/chart_template.py#L691-L692

It probably should if `env.ref` does though